### ISSUE TITLE
Crash with multi-row VALUES

### ISSUE THREAD

**Issue by guofengrichard** [greenplum-db/gpdb#12345]
### Greenplum version or build
6X

### Step to reproduce the behavior

```
# select * from
(values
  (3 not in (select * from (values (1), (2)) ss1)),
  (false)
) ss;
server closed the connection unexpectedly
```

**Comment by jimmyyih** [comment]
This Github issue will be fixed after 9.4_STABLE merge PR is merged:
https://github.com/greenplum-db/gpdb/pull/11784

Specifically, this commit fixes things:
https://github.com/greenplum-db/gpdb/pull/11784/commits/eb9d1f0504a64aeae2b91279bc59e2649d35b4b0

**Comment by adam8157** [comment]
Fixed by the merge PR.
